pylint/extensions/check_elif.py

```diff
@@ -12,7 +12,7 @@ class ElseifUsedChecker(BaseTokenChecker):
     """
 
     __implements__ = (ITokenChecker, IAstroidChecker)
-    name = 'elseifused'
+    name = 'else_if_used'
```

Comment:
FYI, usually we shouldn't change the name, because it might affect pragma disables. For instance, --disable=elseifused would stop working. But in the case of the extensions, it is not that important, since the user already enables the extension explicitly.